### PR TITLE
/trips - trip lines joining markers

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

**Comment by vinyll** [comment]
hooo, I'm loving the idea of this feature ^^
Need to check out the code I guess ;)

File: static/js/trips-map.js [hunk 1]
```diff
@@ -29,19 +34,50 @@ map.fitBounds([[
...
+                    'coordinates': lineCoords
+                }
+            }
+        },
+        'layout': {
+            'line-join': 'round',
+            'line-cap': 'round'
+        },
+        'paint': {
+            'line-color': 'rgba(0, 0, 255, 0.5)',
```
Comment (vinyll) [inline static/js/trips-map.js]: Suggestion for a color and transparency: `rgba(212, 85, 85, 0.3)`.
I tried a few things and this seems to me like a good compromise.

**Review by vinyll — COMMENTED** [review]
Nothing to change. I'm only arguing on the color and its transparency.
Could you check the color I tried and see if that suits you?
For the rest it's all great for me 👍

File: static/js/trips-map.js (hunk 1 above)
Comment (fuentesloic) [reply static/js/trips-map.js]: I think that too

**Pull request merged**